scripts/run_simulator.py

```diff
@@ -36,46 +48,26 @@ def get_webots_parameters() -> tuple[Path, Path]:
     if not world_file.exists():
         raise RuntimeError("World file not found.")
 
+    if not (SIM_BASE / "venv").exists():
+        print("Please run the setup.py script before running the simulator.")
+        raise RuntimeError
```

Comment:
Keeping this to one line will mean the message will be at the bottom of the terminal.
```suggestion
        raise RuntimeError("Please run the setup.py script before running the simulator.")
```